### PR TITLE
Add blog post for 4.3 RC 1

### PR DESCRIPTION
**Pull request by akien-mga**
Socials info:
- TL;DR:
  * Godot 4.3 is ready for mass testing and enters the Release Candidate stage. That means that if no critical bugs are found, this will be the stable release. Everyone interested should consider testing it on (a copy of) their projects so we can make sure the upgrade path from 4.2 is good.
  * Now supports Windows ARM64 devices, and Direct3D 12 is available out of the box.
- URL: https://godotengine.org/article/release-candidate-godot-4-3-rc-1/
- Showcased project: ColdRidge, by Frog Collective :cowboy_hat_face: 
  * Steam: https://store.steampowered.com/app/3052500/ColdRidge/
  * Twitter: https://x.com/FrogCoGames
  * Mastodon: https://mastodon.social/@frogcollective
  * Demo just released on Steam a couple of days ago
  * Picture (from their demo trailer, as the ones on Steam had game logo overlaid):
Screenshot_20240725_155817

### CODE REVIEW

File: collections/_article/release-candidate-godot-4-3-rc-1.md [hunk 1]
```diff
@@ -0,0 +1,121 @@
+---
+title: "Release candidate: Godot 4.3 RC 1"
+excerpt: ""
+categories: ["pre-release"]
+author: "Rémi Verschelde"
+image: /storage/blog/covers/release-candidate-godot-4-3-rc-1.webp
+image_caption_title: "ColdRidge"
+image_caption_description: "A game by Frog Collective"
+date: 2024-07-25 15:00:00
```
Comment (coppolaemilio) [inline collections/_article/release-candidate-godot-4-3-rc-1.md]: We need to start adding the proper time here, but it can be edited after the review. @winston-yallow do you know the local time of the server?

File: collections/_article/release-candidate-godot-4-3-rc-1.md [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+
+Please, consider [supporting the project financially](https://fund.godotengine.org), if you are able. Godot is maintained by the efforts of volunteers and a small team of paid contributors. Your donations go towards sponsoring their work and ensuring they can dedicate their undivided attention to the needs of the project.
+
+[Jump to the **Downloads** section](#downloads), and give it a spin right now, or continue reading to learn more about improvements in this release. You can also [try the **Web editor**](https://editor.godotengine.org/releases/4.3.rc1/) or the **Android editor** for this release. If you are interested in the latter, please request to join [our testing group](https://groups.google.com/g/godot-testers) to get access to pre-release builds.
+
+---
+
+*The cover illustration is from* [**ColdRidge**](https://store.steampowered.com/app/3052500/ColdRidge/-), *a turn-based exploration game set in the mystical Wild West, developed by [Frog Collective](http://frog-collective.com/en/) with Godot 4. The game was revealed in early July and just got a playable demo [on Steam](https://store.steampowered.com/app/3052500/ColdRidge/), where you can also wishlist it.*
```
Comment (coppolaemilio) [inline collections/_article/release-candidate-godot-4-3-rc-1.md]:
```suggestion
*The cover illustration is from* [**ColdRidge**](https://store.steampowered.com/app/3052500/ColdRidge/), *a turn-based exploration game set in the mystical Wild West, developed by [Frog Collective](http://frog-collective.com/) with Godot 4. The game was revealed in early July and just got a playable demo [on Steam](https://store.steampowered.com/app/3052500/ColdRidge/), where you can also wishlist it.*
```

**Review by bruvzg — COMMENTED** [review]
We probably need some download page changes to account for new ARM64 Windows files, something like https://github.com/bruvzg/godot-website/commit/692408dab706aa63fe430265f6a78bf8b702af17 (not tested).

File: collections/_article/release-candidate-godot-4-3-rc-1.md [hunk 3]
```diff
@@ -0,0 +1,121 @@
+---
+title: "Release candidate: Godot 4.3 RC 1"
+excerpt: ""
```
Comment (clayjohn) [inline collections/_article/release-candidate-godot-4-3-rc-1.md]:
```suggestion
excerpt: "We are cautiously optimistic that Godot 4.3 is ready for release, please test it and let us know if we are right!"
```

File: collections/_article/release-candidate-godot-4-3-rc-1.md [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+
+*The cover illustration is from* [**ColdRidge**](https://store.steampowered.com/app/3052500/ColdRidge/-), *a turn-based exploration game set in the mystical Wild West, developed by [Frog Collective](http://frog-collective.com/en/) with Godot 4. The game was revealed in early July and just got a playable demo [on Steam](https://store.steampowered.com/app/3052500/ColdRidge/), where you can also wishlist it.*
+
+## Highlights
+
+We covered the most important highlights from Godot 4.3 in the previous [4.3 beta 1 blog post](/article/dev-snapshot-godot-4-3-beta-1/), so if you haven't read that one, have a look to be introduced to the main new features added in the 4.3 release.
+
+Especially if you're testing 4.3 for the first time, you'll want to get a condensed overview of what new features you might want to make use of.
+
+This section covers changes made since the previous [beta 3 snapshot](/article/dev-snapshot-godot-4-3-beta-3/), which are mostly regression fixes, or "safe" fixes to long standard issues.
```
Comment (clayjohn) [inline collections/_article/release-candidate-godot-4-3-rc-1.md]:
```suggestion
This section covers changes made since the previous [beta 3 snapshot](/article/dev-snapshot-godot-4-3-beta-3/), which are mostly regression fixes, or "safe" fixes to longstanding issues.
```

File: collections/_article/release-candidate-godot-4-3-rc-1.md [hunk 5]
```diff
@@ -0,0 +1,121 @@
...
+
+There are two big highlights in this release for the Windows platform though:
+
+- The Direct3D 12 rendering backend now works out of the box on Windows, and no longer requires copying the `DXIL.dll` library ([GH-94203](https://github.com/godotengine/godot/pull/94203))! This is thanks to Microsoft's decision to [open source the DXIL Validator hash](https://devblogs.microsoft.com/directx/open-sourcing-dxil-validator-hash/), which was the only reason for us to require the proprietary `DXIL.dll`. Aside from greatly simplifying the distribution of Windows builds with D3D12, it also makes it possible to export Windows D3D12 projects from other OSes (Linux, macOS, Android), which is an important part of Godot's cross-platform support.
+- With the progressive rollout of ARM64 Windows devices on the market, we've finalized our support for that target and now provide ARM64 Windows builds (editor and export templates) for the first, for both Standard and .NET flavors. Thanks to Qualcomm engineers who did some testing and gave us access to cloud-based devices to confirm that the binaries work.
```
Comment (clayjohn) [inline collections/_article/release-candidate-godot-4-3-rc-1.md]:
```suggestion
- With the progressive rollout of ARM64 Windows devices on the market, we've finalized our support for that target and now provide ARM64 Windows builds (editor and export templates) for both Standard and .NET flavors. Thanks to Qualcomm engineers who did some testing and gave us access to cloud-based devices to confirm that the binaries work.
```

**Review by clayjohn — COMMENTED** [review]
A few suggestions. The rest looks good to me

File: collections/_article/release-candidate-godot-4-3-rc-1.md (hunk 1 above)
Comment (akien-mga) [reply collections/_article/release-candidate-godot-4-3-rc-1.md]: Is it important for the time to be accurate? We've always picked roughly a time that's before publishing date on the same day, which seems good enough. I guess mostly RSS feeds might see it as "published 3 hours ago" even though it was published now.

As for local time, I'm assuming UTC here.

Still, if we can infer it at build time, that could work. But since we rebuild everything on each commit, we'd need that information to be committed somehow I guess?

**Comment by akien-mga** [comment]
> We probably need some download page changes to account for new ARM64 Windows files, something like [bruvzg@692408d](https://github.com/bruvzg/godot-website/commit/692408dab706aa63fe430265f6a78bf8b702af17) (not tested).

That looks great!

As we discussed on Rocket.Chat, for RC1 we won't update the website and just soft-launch the arm64 version. I've also re-released it with the `Godot_v4.3-rc1_windows_arm64` base name instead of `Godot_v4.3-rc1_arm64` which lacked the platform.

For RC2/stable, I'll review all binary names for consistency and we can update the website then to include arm64 on Windows, and update other patterns if they changed. (Maybe also get rid of the "64-bit"/"32-bit" tags and always explicitly use the architecture?)

**Comment by akien-mga** [comment]
Added some direct links to the Windows ARM64 builds in the download section.

**Comment by akien-mga** [comment]
Thanks for the reviews!
